src/aui-form-builder/js/aui-form-builder.js

```diff
@@ -544,6 +513,37 @@ A.FormBuilder = A.Base.create('form-builder', A.Widget, [
     },
 
     /**
+     * Form Builder Pages instance initializer. Receives a custom
+     * object of options or using default options instead.
+     * 
+     * @method _getPagesInstance
+     * @param {Object} options
```

Comment:
This should be "config" instead of "options", since that's the pattern for this kind of situation, when you need create a new component using the A.Merge.